<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <widget class="QPushButton" name="hostButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIpLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter server IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostIpLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Host IP: </string>
    </property>
   </widget>
   <!-- Game board buttons -->
   <widget class="QPushButton" name="button00">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button01">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button02">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button22">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

